HW2.Rmd

```diff
@@ -0,0 +1,91 @@
+---
+title: "Crazy Super Awesome HW2"
+author: "Izzy Gaw"
+date: "2020-01-22"
+output: html_document
+---
+
+<html>
+<body bgcolor="green">
+<html>
+
+```{r}
+#########
+###HW2###
+#########
+
+read.csv("iris.csv") #read in data set
```

Comment:
Hey Izzy, this is a very nice script but it is written in the format of an R script, and not taking advantage of the formatting in markdown. 

Remember to put your narrative in paragraph form (or just sentences) above the blocks of code. For example:

# HW2

Read in the dataset and rename variables. Iʻll analyze sepal and petal lengths, by regression with sepal as the x variable and petal as the y:
```{r}
iris <- read.csv("iris.csv")
names(iris) <- c("sepalL", "sepalW",  "petalL", "petalW", "species")
x <- iris$sepalL
y <- iris$petalL
```
More narrative
```{r}
another code block
```
etc.